website/docusaurus.config.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// See https://docusaurus.io/docs/site-config for all the possible
+// site configuration options.
+
+const siteConfig = {
+	title: "Redex", // Title for your website.
+	tagline: "An Android Bytecode Optimizer",
+	url: "https://fbredex.com",
+	baseUrl: "/",
+
+	// Used for publishing and more
+	organizationName: "facebook",
+	projectName: "redex",
+
+	favicon: "img/favicon.png",
+
+	/* Colors for website */
+	// colors: {
+	//   primaryColor: '#75c7a4',
+	//   secondaryColor: '#f9f9f9',
+	// },
+
+	/* Custom fonts for website */
+	/*
+  fonts: {
+    myFont: [
+      "Times New Roman",
+      "Serif"
+    ],
+    myOtherFont: [
+      "-apple-system",
+      "system-ui"
+    ]
+  },
+  */
+
+	// highlight: {
+	// 	// Highlight.js theme to use for syntax highlighting in code blocks.
+	// 	theme: "default"
+	// },
+
+	// Add custom scripts here that would be placed in <script> tags.
+	scripts: ["https://buttons.github.io/buttons.js"],
```

Comment:
i dont think its needed now

Reply:
So github stars won't be coming,in v1 it was present in footer https://fbredex.com/

Reply:
Yep we don't plan to show it in the footer anymore. You may ignore that.